envoy/stream_info/stream_info.h

```diff
@@ -240,6 +240,16 @@ struct UpstreamTiming {
 
 class DownstreamTiming {
 public:
+  void setValue(absl::string_view key, MonotonicTime value) { timings_[key] = value; }
```

Comment:
Should we assert that we don't set a value twice? Not sure.

Reply:
yeah I didn't know if folks using this would want to do updates, so left without.